webpack/webpack-embedded-dependencies-plugin/src/EmbeddedDependenciesWebpackPlugin.ts

```diff
@@ -0,0 +1,387 @@
+import path from 'path';
+import { Compilation } from 'webpack';
```

Comment:
Consolidate as a type-only import.

Reply:
This is required for `Compilation.PROCESS_ASSETS_STAGE_REPORT`

Reply:
`compiler.webpack.Compilation.PROCESS_ASSETS_STAGE_REPORT`